TYPO3.Fluid/Tests/Functional/Core/ParserIntegrationTest.php

```diff
@@ -94,4 +95,27 @@ public function layoutViewHelperCanContainIfCondition()
         $this->assertSame($uncompiledResult, $compiledResult, 'The rendered compiled template did not match the rendered uncompiled template.');
         $this->assertSame('foo: Content', $standaloneView->render());
     }
+
+    /**
+     * @test
+     */
+    public function isHasMethodsCanBeAccessedDirectly()
+    {
+        $httpRequest = Request::create(new Uri('http://localhost'));
+        $actionRequest = new \TYPO3\Flow\Mvc\ActionRequest($httpRequest);
+
+        $standaloneView = new StandaloneView($actionRequest, uniqid());
+        $post = new Post();
+        $post->setPrivate(true);
+        $standaloneView->assignMultiple(array('post' => $post));
+        $standaloneView->setTemplateSource('<f:if condition="{post.isPrivate}">Private!</f:if>');
+
+        $actual = $standaloneView->render();
+        $this->assertSame('Private!', $actual);
+
+        $standaloneView->setTemplateSource('<f:if condition="{post.isprivate}">Private!</f:if>');
+
+        $actual = $standaloneView->render();
+        $this->assertSame('', $actual);
```

Comment:
Great, that you added a test, even a functional one!
Just a mini nit-pick: If you would extract the second case ("post.isprivate") into its own test method it would be easier to read & debug in case of errors